bin/build-docker.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+/**
+ * Script to run docker build.
+ *
+ * This script is needed to enable us to embed the git SHA as a build arg
+ * in both unix shells and windows.
+ *
+ * Essentially, it doees this:
+ * docker build --build-arg commit_sha=`git rev-parse HEAD` -t wp-calypso .
+ */
+
+const { promisify } = require( 'util' );
+const { spawn, exec } = require( 'child_process' );
+
+function readSha() {
+	return promisify( exec )( 'git rev-parse HEAD' ).then( ( { stdout } ) => {
```

Comment:
What’s the point of using promisify here?

Reply:
Just syntactic sugar for clarity of the code below, so I didn't have to pass in a function to `readSha()` or put inline code inside it to call `docker build`.

Reply:
that's what I thought. seemed strange to pull it in here just to change the calling syntax of the function (vs. passing a callback). I don't care too much.

Reply:
All this would be easier using `spawnSync` and `execSync`, this is a sequential script after all.

Reply:
I didn't even notice that's what we're doing; those are great options.

Reply:
Updated to use sync functions.